### PR TITLE
DCA-35: Add application manager role to D&T dev and DCA Prod accounts

### PR DESCRIPTION
**Pull request by brucehoff**
This PR defines the 'application manager' roles which lets the creator of an application deployed to AWS monitor it and set up secrets.  The PR also deploys the role to the DnT dev and DCA prod accounts

### CODE REVIEW

File: org-formation/700-aws-sso/_tasks.yaml [hunk 1]
```diff
@@ -422,6 +430,29 @@ SsoViewerPlus:
...
+  StackName: !Sub '${resourcePrefix}-${appName}-application-manager'
+  StackDescription: 'Read-only role plus access to secrets manager'
+  TerminationProtection: false
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account: '*'
+  Parameters:
```
Comment (zaro0508) [inline org-formation/700-aws-sso/_tasks.yaml:446]: `principalId` is a required parameter which means you'll need to create an AWS SSO group for this as well.  the group can be empty.

File: org-formation/700-aws-sso/_tasks.yaml [hunk 2]
```diff
@@ -1347,6 +1378,23 @@ SsoDntDevViewer:
     principalId: !Ref dntDevViewerGroup
     permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-viewer-permission-set-arn' ]
 
+SsoDntDevProdApplicationManager:
+  Type: update-stacks
+  DependsOn: SsoApplicationManager
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.3.8/templates/SSO/aws-sso.yaml
```
Comment (zaro0508) [inline org-formation/700-aws-sso/_tasks.yaml:1384]: should probably use the newer version `https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.1/templates/SSO/aws-sso.njk`

File: org-formation/700-aws-sso/_tasks.yaml [hunk 3]
```diff
@@ -1500,6 +1548,23 @@ SsoDCAProdViewerPlus:
     principalId: !Ref dcaProdViewersPlusGroup
     permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-viewer-plus-permission-set-arn' ]
 
+SsoDCAProdApplicationManager:
+  Type: update-stacks
+  DependsOn: SsoApplicationManager
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.3.8/templates/SSO/aws-sso.yaml
```
Comment (zaro0508) [inline org-formation/700-aws-sso/_tasks.yaml:1554]: newer version

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: org-formation/700-aws-sso/_tasks.yaml (hunk 1 above)
Comment (brucehoff) [reply org-formation/700-aws-sso/_tasks.yaml:446]: OK to just use `adminGroup`? This avoids creating an empty placeholder.

File: org-formation/700-aws-sso/_tasks.yaml (hunk 2 above)
Comment (brucehoff) [reply org-formation/700-aws-sso/_tasks.yaml]: Done.

File: org-formation/700-aws-sso/_tasks.yaml (hunk 3 above)
Comment (brucehoff) [reply org-formation/700-aws-sso/_tasks.yaml]: Done.

File: org-formation/700-aws-sso/_tasks.yaml (hunk 1 above)
Comment (zaro0508) [reply org-formation/700-aws-sso/_tasks.yaml:446]: good idea.

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**